examples/activities/resample.js

```diff
@@ -62,19 +69,28 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
         return done.apply(null, arguments);
       }
 
-      var ds = gdal.open(localOutputPath);
-
-      try {
-        ds.bands.get(1).getStatistics(false, true);
-      } catch (err) {
-        // likely "Failed to compute statistics, no valid pixels found in
-        // sampling.", which means that it's all NODATA, and thus not worth
-        // saving
-
-        return done();
-      } finally {
-        ds.close();
-      }
+      // try {
```

Comment:
I found that `gdal2tilesp.py` doesn't do this either (which could account for some of the speed-up).  It'd be nice to figure out a way to filter out ND tiles after-the-fact, but this seems like a reasonable solution.